yt_dlp/extractor/massengeschmacktv.py

```diff
@@ -30,7 +31,7 @@ def _real_extract(self, url):
 
         webpage = self._download_webpage(url, episode)
         title = clean_html(self._html_search_regex(
-            '<h3>([^<]+)</h3>', webpage, 'title'))
+            '<[^>]+(?:id|ID)=["\']clip-title["\'][^>]*>([^<]+)', webpage, 'title'))
```

Comment:
IMO it would be better to use `get_element_by_id` (import from utils)

And we want it to be non-fatal